packages/react-graphql/src/hooks/query.ts

```diff
@@ -35,10 +35,14 @@ export default function useQuery<
     client: overrideClient,
     notifyOnNetworkStatusChange,
     context,
+    ssr = true,
   } = options;
   const client = useApolloClient(overrideClient);
 
-  if (typeof window === 'undefined' && (skip || fetchPolicy === 'no-cache')) {
+  if (
+    typeof window === 'undefined' &&
+    (skip || fetchPolicy === 'no-cache' || !ssr)
+  ) {
     return {...createDefaultResult(client, variables), loading: !skip};
```

Comment:
Do you know what the behaviour of `loading` should be here?

Reply:
loading can be infer from here
https://github.com/trojanowski/react-apollo-hooks/blob/master/src/useQuery.ts#L89

Reply:
Good point.
Looks like [they are setting is to `false` during SSR ](https://github.com/trojanowski/react-apollo-hooks/blob/master/src/useQuery.ts#L160)(`react-apollo-hook`)

Would it be safe to always set `loading` to `false`?

Reply:
I guess `loading = false` probably makes sense for any kind of skipping, yeah

Reply:
false is !ssr? https://github.com/trojanowski/react-apollo-hooks/blob/master/src/useQuery.ts#L89